english/dream/dream.md

```diff
@@ -0,0 +1,9 @@
+You understand that it's difficult to wake up from a dream if you think this is all really happening to you.
+
+You attempt to point out things that are around you that cannot possibly occur in real life such the marshmallow walls and hand.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.